Cargo.toml

```diff
@@ -114,20 +115,19 @@ mimalloc = "0.1.42"
 moka = { version = "0.12.10", default-features = false }
 num-traits = "0.2.19"
 num_enum = "0.7.2"
-object_store = "0.11.0"
+object_store = "0.12.0"
 opentelemetry = "0.29.0"
 opentelemetry-otlp = "0.29.0"
 opentelemetry_sdk = "0.29.0"
 parking_lot = "0.12.3"
-parquet = "54.3.1"
+parquet = "55"
 paste = "1.0.15"
 pin-project = "1.1.5"
 pin-project-lite = "0.2.15"
 prost = "0.13.4"
 prost-build = "0.13.4"
 prost-types = "0.13.4"
-# bump pyo3 to >0.24 on next arrow release, for RUSTSEC-2025-0020
-pyo3 = { version = "0.23.4", features = ["extension-module", "abi3-py310"] }
+pyo3 = { version = "0.24.1", features = ["extension-module", "abi3-py310"] }
```

Comment:
with this we can also remove the ignore in `deny.toml`

Reply:
done